client/src/pages/UserLogin/UserLogin.js

```diff
@@ -21,7 +19,7 @@ const UserLogin = () => {
       }
     );
     const data = response.data;
-    setState({ token: data.token });
+    sessionStorage.setItem('loginToken', data.token);
```

Comment:
create a utils folder and put this there, kay future proofing ba what if naa natay feature nga sigeg access sa sessionstorage? so create a file lang for this and maybe call it `authSessionStorage` nya create a function for this nga maybe `setLoginToken` nya respectively for getting kay `getLoginToken`

Reply:
resolved